src/software/jetson_nano/services/motor.cpp

```diff
@@ -195,13 +191,15 @@ bool MotorService::checkDriverFault(uint8_t motor)
         LOG(WARNING)
             << "Indicates that the IC has been reset. All registers have been cleared to reset values."
             << "Attention: DRV_EN must be high to allow clearing reset";
+        motor_faults.insert(TbotsProto::MotorFault::RESET);
```

Comment:
nima: The one thing I've previously noticed is that the RESET bit always seems to be set to 1. So if we're frequently checking for faults, it will keep spamming the "==== FAULTS FOR MOTOR ..." log and this log. I'm not really familiar with why this bit is always set and if that's expected. If it is expected, then maybe we should remove their logs to reduce the spam